### PR TITLE
Tweaking

### PR DESCRIPTION
**Pull request by RDIL**
Remove unused stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,24 +12,12 @@ searching the web but have these things be built into your terminal.
...
-```bash
-sudo make install_deps
-```
+Use Gitpod, everything is preinstalled.
 
 ## Install
 
 ### Build from source
 
-#### Install with setup script
```
Comment (JesterOrNot) [inline README.md]: We need users to know how to build locally

Comment (RDIL) [reply README.md]: This is why we have make

Comment (JesterOrNot) [reply README.md]: Fair enough

File: .gitpod.yml [hunk 2]
```diff
@@ -1,18 +1,18 @@
 image:
   file: .gitpod.Dockerfile
+
 tasks:
-  - init: mkdir ../go ../go/bin && dh_auto_install
-    command: gosh
+  - init:  mkdir ../go ../go/bin && dh_auto_install
```
Comment (JesterOrNot) [inline .gitpod.yml]: Why are we adding setup.sh to command?

**Pull request merged**